packages/cli-kit/src/analytics.test.ts

```diff
@@ -61,11 +64,12 @@ describe('event tracking', () => {
         storeFqdn: 'domain1',
         directory: '/cached',
       })
-      start({command, args, currentTime: currentDate.getTime() - 100})
+      await start({command, args, currentTime: currentDate.getTime() - 100})
 
       // When
       const config = {
         runHook: vi.fn().mockResolvedValue({successes: [], failures: []}),
+        plugins: [],
```

Comment:
should we add some plugins to the test to validate that they are added to the payload?